### PR TITLE
add and modified readme for Traditional Chinese.

### PR DESCRIPTION
**Pull request by noahcse**
hi, I join PyConTW sprints pygame session, and wanna to make a traditional Chinese doc for people who use traditional Chinese to collab and enjoy fun of pygame.

### CODE REVIEW

**Review by ankith26 — APPROVED** [review]
I don't know Traditional Chinese, but I am approving this PR nonetheless. I did a quick check with google translate and found nothing objectionable.

Thanks for contributing! :partying_face:

**Comment by bilhox** [comment]
Now that japanese translation was merged, it's time to fix merge conflicts by updating your branch.
Anyways, thank you for the PR!

**Review by yunline — DISMISSED** [review]

File: docs/readmes/README.zh-tw.rst [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
++----------+------------------------+
+| CPython  | >= 3.8 （或 PyPy3）    |
++----------+------------------------+
+| SDL      | >= 2.0.10              |
++----------+------------------------+
+| SDL_mixer| >= 2.0.4               |
++----------+------------------------+
+| SDL_image| >= 2.0.4               |
++----------+------------------------+
+| SDL_ttf  | >= 2.0.15              |
```
Comment (MyreMylar) [inline docs/readmes/README.zh-tw.rst:121]:
```suggestion
| CPython  | >= 3.9 （或 PyPy3）    |
+----------+------------------------+
| SDL      | >= 2.0.14              |
+----------+------------------------+
| SDL_mixer| >= 2.0.4               |
+----------+------------------------+
| SDL_image| >= 2.0.4               |
+----------+------------------------+
| SDL_ttf  | >= 2.0.15              |
```

File: docs/readmes/README.zh-tw.rst [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
++----------+------------------------+
+
+
```
Comment (MyreMylar) [inline docs/readmes/README.zh-tw.rst:124]: Missing a translation of the new 'How to contribute' section which currently reads:

> How to Contribute
> -----------------
> First of all, thank you for considering contributing to pygame-ce! It's people like you that make pygame-ce a great library. Please follow these steps to get started:
> 
> 1. Read the `Contribution Guidelines`_ and the `Many Ways to Contribute`_ wiki pages.
> 2. Read the documentataion on `Opening A Pull Request`_ and `Opening a Great Pull Request`_.
> 3. Read how to `label and link reported issues`_.
> 4. Check the `issue tracker`_ for open issues that interest you or open a new issue to start a discussion about your idea.
> 
> There are many more resources throughout the `wiki pages`_ that can help you get started.
> 
> If you have any questions, please feel free to ask in the `Pygame Community Discord Server`_ or open an issue.

File: docs/readmes/README.zh-tw.rst [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+感謝製作遊戲並把遊戲放到pygame.org網站供人學習與娛樂的人。
+
+感謝 James Paige 建立了pygame bugzilla。
+
+感謝 Roger Dingledine 與SEUL.ORG上的crew，感謝我們優秀的主持。
+
+依賴
+------------
+
+**note:** This section translation is out of date (September 2023)
```
Comment (MyreMylar) [inline docs/readmes/README.zh-tw.rst:104]:
```suggestion
```

Lets just delete this line and update the section.

File: docs/readmes/README.zh-tw.rst [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+Michael George, Saul Spatz, Thomas Ibbotson, Tom Rothamel, Evan Kroske,
+Cambell Barton.
+
+以及我們卓越的bug獵人：Angus, Guillaume Proux, Frank
+Raiser, Austin Henry, Kaweh Kazemi, Arturo Aldama, Mike Mulcheck,
+Michael Benfield, David Lau
+
+還有許多人提交了有用的想法，助力本計畫前進，使我們的生活變得更加輕鬆。感謝你們！
+
+感謝對文檔做成評論並將其添加到pygame.org wiki的人。
```
Comment (MyreMylar) [inline docs/readmes/README.zh-tw.rst:93]: this line has changed in the English readme and should be translated:

> Many thank you's for people making documentation comments, and adding to the
> `pygame documentation`_ and the `pygame-ce documentation`_.

**Review by MyreMylar — COMMENTED** [review]
Noticed a couple of missed sections.

I'll check the other chinese readme PR and see if they are included.

File: docs/readmes/README.zh-tw.rst [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+| SDL      | >= 2.0.14              |
++----------+------------------------+
+| SDL_mixer| >= 2.0.4               |
++----------+------------------------+
+| SDL_image| >= 2.0.4               |
++----------+------------------------+
+| SDL_ttf  | >= 2.0.15              |
++----------+------------------------+
+
+
```
Comment (MyreMylar) [inline docs/readmes/README.zh-tw.rst:123]:
```suggestion

如何貢獻
--------

首先，感謝你考慮為 pygame-ce 做出貢獻！正是像你這樣的人讓 pygame-ce 成為一個偉大的函式庫。請依照以下步驟開始：

1. 閱讀 `Contribution Guidelines`_ 和 `Many Ways to Contribute`_ wiki 頁面。
2. 閱讀 `Opening A Pull Request`_ 和 `Opening a Great Pull Request`_ 的相關文件。
3. 學習How to `label and link reported issues`_ 。
4. 檢查 `issue tracker`_ 以查找你有興趣的issue，或建立一個新的issue來開始討論你的想法。

`wiki pages`_ 上還有許多資源可以幫助你入門。

如果有任何問題，請隨時在 `Pygame Community Discord Server`_ 詢問或建立一個issue。

```

File: docs/readmes/README.zh-tw.rst [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+Michael George, Saul Spatz, Thomas Ibbotson, Tom Rothamel, Evan Kroske,
+Cambell Barton.
+
+以及我們卓越的bug獵人：Angus, Guillaume Proux, Frank
+Raiser, Austin Henry, Kaweh Kazemi, Arturo Aldama, Mike Mulcheck,
+Michael Benfield, David Lau
+
+還有許多人提交了有用的想法，助力本計畫前進，使我們的生活變得更加輕鬆。感謝你們！
+
+感謝對文檔做成評論並將其添加到pygame.org wiki的人。
```
Comment (MyreMylar) [inline docs/readmes/README.zh-tw.rst:93]:
```suggestion
非常感謝人們提出文件評論並添加到 pygame 文件和 pygame-ce 文檔
```

**Review by MyreMylar — APPROVED** [review]
OK, I think that is the main updates and merges the two PRs.

File: docs/readmes/README.es.rst [hunk 7]
```diff
@@ -8,7 +8,6 @@
 |Python3| |GithubCommits| |BlackFormatBadge|
 
 `English`_ `简体中文`_ `繁體中文`_ `Français`_ `فارسی`_ **Español** `日本語`_ `Italiano`_ `Русский`_
```
Comment (MyreMylar) [inline docs/readmes/README.es.rst:10]:
```suggestion

`English`_ `简体中文`_ `繁體中文`_ `Français`_ `فارسی`_ **Español** `日本語`_ `Italiano`_ `Русский`_

```

File: docs/readmes/README.fa.rst [hunk 8]
```diff
@@ -8,7 +8,6 @@
 |Python3| |GithubCommits| |BlackFormatBadge|
 
 `English`_ `简体中文`_ `繁體中文`_ `Français`_ **فارسی** `Español`_ `日本語`_ `Italiano`_ `Русский`_
```
Comment (MyreMylar) [inline docs/readmes/README.fa.rst:10]:
```suggestion
`English`_ `简体中文`_ `繁體中文`_ `Français`_ **فارسی** `Español`_ `日本語`_ `Italiano`_ `Русский`_

```

File: docs/readmes/README.fr.rst [hunk 9]
```diff
@@ -8,7 +8,6 @@
 |Python3| |GithubCommits| |BlackFormatBadge|
 
 `English`_ `简体中文`_ `繁體中文`_ **Français**  `فارسی`_ `Español`_ `日本語`_ `Italiano`_ `Русский`_
```
Comment (MyreMylar) [inline docs/readmes/README.fr.rst:10]:
```suggestion
`English`_ `简体中文`_ `繁體中文`_ **Français**  `فارسی`_ `Español`_ `日本語`_ `Italiano`_ `Русский`_

```

File: docs/readmes/README.zh-cn.rst [hunk 10]
```diff
@@ -8,7 +8,6 @@
 |Python3| |GithubCommits| |BlackFormatBadge|
 
 `English`_ **简体中文** `繁體中文`_ `Français`_ `فارسی`_ `Español`_ `日本語`_ `Italiano`_ `Русский`_
```
Comment (MyreMylar) [inline docs/readmes/README.zh-cn.rst:10]:
```suggestion
`English`_ **简体中文** `繁體中文`_ `Français`_ `فارسی`_ `Español`_ `日本語`_ `Italiano`_ `Русский`_

```

**Pull request merged**